content/blog/2023-07-06-oreoscan-spotlight.mdx

```diff
@@ -0,0 +1,65 @@
+---
+author: elena
+title: "Ecosystem Spotlight: Oreoscan"
+description: Learn more about Oreoscan, an incredible community member who's building on the Iron Fish network
+image: /images/ecosystem/oreoscan-spotlight.png
```

Comment:
```suggestion
image: /images/ecosystem/oreoscan.png
```

![Screen Shot 2023-07-06 at 12 45 52 PM](https://github.com/iron-fish/website/assets/5459049/9ad43566-ec14-420d-8d0b-8dd30a80fd2f)

Is this image correct?